utils/apiUtil.js

```diff
@@ -139,6 +140,22 @@ module.exports = {
     };
   },
 
+  getJWTData(path_to_credentials_json) {
```

Comment:
Let us put some documentation for this function (just like we have for the other API Util ones):
```
/**
   * Reads the credentials.json file, parse & filter its contents to return the api_key and the target_environment
   * @param {file path} - the path to the credentials.json file (downloaded from a given Gen3 Commons)
   * @returns a tuple {string, string}
   */
```